src/test/java/core/basesyntax/FileReaderTest.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
```

Comment:
It's a good practice to create same folders structure for tests as for classes they test.
If you FileReaderImpl is in folder `core.basesyntax.service.impl` then test should be also in forder 'service.impl' in test package.
Ans similar for other classes